packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -59,58 +58,118 @@ const NominationBanner = (props) => {
       (declineAppModalVisible) => !declineAppModalVisible
     );
   };
-
+  function attachmentsHandled() {
+    const value = false;
+    activeNomination.attachments = value;
+    setActiveNomination({...activeNomination})
```

Comment:
Instead of assigning the variable separately, you can use the spread operator like this `setActiveNomination({...activeNomination, attachments: false})`

Reply:
changed